modules/openy_features/openy_media/config/install/image.style.full.yml

```diff
@@ -0,0 +1,14 @@
+langcode: en
```

Comment:
According to naming convention, image style should have prefix of entity `image.style.media_full.yml` and label "Media full".